From b6b45f698846df59fa2a4f58abd5d01bdcae2f35 Mon Sep 17 00:00:00 2001
From: Zhou Furong <furong.zhou@intel.com>
Date: Wed, 14 Nov 2018 09:13:11 +0800
Subject: [PATCH 2290/2345] Add fs header files to fix compile errors on
 m68k/sh

m68k ans sh need this header files to pass compilation.

Change-Id: I45fe0b10cb37d874344f140ae5c827d7222593e4
---
 drivers/char/rpmb/mux/mux.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/char/rpmb/mux/mux.c b/drivers/char/rpmb/mux/mux.c
index c1b5d37..69f47cd 100644
--- a/drivers/char/rpmb/mux/mux.c
+++ b/drivers/char/rpmb/mux/mux.c
@@ -6,6 +6,7 @@
  */
 
 #include <linux/module.h>
+#include <linux/fs.h>
 #include <linux/compat.h>
 #include <linux/init.h>
 #include <linux/kernel.h>
-- 
2.7.4

